pkg/api/service/testing/make.go

```diff
@@ -160,3 +160,17 @@ func SetInternalTrafficPolicy(policy api.ServiceInternalTrafficPolicyType) Tweak
 		svc.Spec.InternalTrafficPolicy = &policy
 	}
 }
+
+// SetExternalTrafficPolicy sets the externalTrafficPolicy field for a Service.
+func SetExternalTrafficPolicy(policy api.ServiceExternalTrafficPolicyType) Tweak {
+	return func(svc *api.Service) {
+		svc.Spec.ExternalTrafficPolicy = policy
+	}
+}
+
+// SetAllocateLoadBalancerNodePorts sets the allocate LB node port field.
+func SetAllocateLoadBalancerNodePorts(val bool) Tweak {
+	return func(svc *api.Service) {
+		svc.Spec.AllocateLoadBalancerNodePorts = utilpointer.BoolPtr(val)
+	}
+}
```

Comment:
these doesn't seems to be used in any other place, is a leftover from other branch or something you are planning to add?

Reply:
the unit test is failing because of this, so I think is missing below

```
    rest_test.go:633: Expected no error: Service "foo" is invalid: allocateLoadBalancerNodePorts: Required value
```

Reply:
I needed to rebase this on https://github.com/kubernetes/kubernetes/pull/103467

Reply:
done in new push